daffodil-core/src/main/scala/org/apache/daffodil/dsom/CompiledExpression.scala

```diff
@@ -23,6 +23,7 @@ import scala.xml.NamespaceBinding
 import org.apache.daffodil.xml.NamedQName
 import java.lang.{ Long => JLong, Boolean => JBoolean }
 import org.apache.daffodil.schema.annotation.props.Found
+import org.apache.daffodil.util.DPathUtil
```

Comment:
Is this import really used?  Is there an `import org.apache.daffodil.dpath.DPathUtil` that should be removed?

Reply:
The DPathUtil import is used for the new call to DPathUtil.isExpression on line 64.